.github/workflows/docker.yml

```diff
@@ -0,0 +1,23 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-docker:
+
+      runs-on: ubuntu-latest
+
+      steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: docker-compose build
+
+      - name: Run
+        run: docker-compose run application
```

Comment:
No newlines at the end of the file really make me feel uneasy.

Reply:
Done!